File: beginner/1134.py
alcool = 0
gasolina = 0
diesel = 0

while True:
    aux = int(input())

    if aux == 1:
        alcool += 1
    if aux == 2:
        gasolina += 1
    if aux == 3:
        diesel += 1
    if aux == 4:
        break

print("MUITO OBRIGADO")
print("Alcool:", alcool)
print("Gasolina:", gasolina)
print("Diesel:", diesel)

File: beginner/1134.c
# include <stdio.h>

int main(){

    int aux, alcool, gasolina, diesel;
    aux = alcool = gasolina = diesel = 0;

    while(1){
        scanf("%i", &aux);

        if( aux == 1 ){
            alcool++;
        }

        if( aux == 2 ){
            gasolina++;
        }

        if( aux == 3 ){
            diesel++;
        }

        if( aux == 4 ){
            break;
        }
    }

    printf("MUITO OBRIGADO\n");
    printf("Alcool: %i\n", alcool);
    printf("Gasolina: %i\n", gasolina);
    printf("Diesel: %i\n", diesel);

    return 0;
}